src/window.rs

```diff
@@ -242,6 +244,7 @@ impl Window {
     }
 
     /// Set cursor visibility
+    #[deprecated(note = "Use `set_cursor_state` instead")]
```

Comment:
Let's remove it! We're not caring much about backward compatibility at the moment.